tests/test_request.py

```diff
@@ -30,7 +31,7 @@ def test():
 
     with app.test_client() as client:
         # No content-type, no max content length checking
-        data = b"a" * (max_len + 1)
+        data = b"a" * (max_len - 1)
```

Comment:
This one is strange... Maybe the test was bad/faulty from before? Or did the test client change maybe, and now the `conten-type` is passed implicitly?

Reply:
as i understand the [PR](https://github.com/pallets/werkzeug/commit/4f7048e7a31752142f18eefeccd49acc42a89e31#diff-f94258ec650ac61110aed42f4708d43590a6a3e8ebb76234d42a933c3796886d) the thing is that the max_content_length hasn't been checked before.

now it is checked.